src/DKIMMailServiceProvider.php

```diff
@@ -35,7 +35,11 @@ private function publishesConfig(): void
     protected function registerIlluminateMailer()
     {
         $this->app->singleton('mail.manager', static function (Application $app) {
-            return new MailManager($app);
+            if (!is_null(config('dkim.domain'))) {
+                return new MailManager($app);
+            } else {
```

Comment:
because there is a return above, we can leave out the else and just return the other MailManager below.